.teamcity/src/main/kotlin/configurations/FlakyTestQuarantine.kt

```diff
@@ -15,12 +16,12 @@ import model.CIBuildModel
 import model.Stage
 import model.StageNames
 
-class FlakyTestQuarantine(model: CIBuildModel, stage: Stage, os: Os) : BaseGradleBuildType(stage = stage, init = {
-    id("${model.projectId}_FlakyQuarantine_${os.name.lowercase().toCapitalized()}")
-    name = "Flaky Test Quarantine - ${os.name.lowercase().toCapitalized()}"
+class FlakyTestQuarantine(model: CIBuildModel, stage: Stage, os: Os, arch: Arch = Arch.AMD64) : BaseGradleBuildType(stage = stage, init = {
+    id("${model.projectId}_FlakyQuarantine_${os.asName()}_${arch.name}")
```

Comment:
So we'd lose the history here. Do you think it would make sense to use numbers for the `FlakyTestQuarantine` "buckets" as well?

Reply:
I expect we won't change the id frequently after this change so i think it's good right now.